cms/static/js/features_jsx/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
@andy-armstrong and @arizzitano I'm still getting jslint errors on my React file even after including this file (which I copied from the one in course_experience).

@andy-armstrong, we do plan to use the jsx extension soon, but in the very short term planned to check es2015 files into their own directory.